ls1intum/Artemis
# [Bugfix] fixing the "Server Administration>API" subpage

### Checklist

### Description

Instead of having the OpenAPi embedded, i now open a new Tab with the OpenAPI started

### Steps for Testing

1. Login to Artemis with an admin/Instructor
2. Navigate to Server Administration/API and verify it opens a new Tab with the SwaggerUI loading...
3. The Performance of the OpenAPI still sucks, it is simply due to the massive amount of endpoints we provide; Sometimes it also times out while loading
